File: app/src/main/java/me/ccrama/redditslide/Activities/SwipeTutorial.java
package me.ccrama.redditslide.Activities;

import android.os.Bundle;
import android.widget.TextView;

import ltd.ucode.slide.R;


/**
 * Created by ccrama on 3/5/2015.
 */

public class SwipeTutorial extends BaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.swipe_tutorial);
        if(getIntent().hasExtra("subtitle")){
            ((TextView)findViewById(R.id.subtitle)).setText(getIntent().getStringExtra("subtitle"));
        }
    }
}

File: app/src/main/java/me/ccrama/redditslide/Activities/SwipeTutorial.kt
package me.ccrama.redditslide.Activities

import android.os.Bundle
import android.view.View
import android.widget.TextView
import ltd.ucode.slide.R

class SwipeTutorial : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.swipe_tutorial)
        if (intent.hasExtra("subtitle")) {
            (findViewById<View>(R.id.subtitle) as TextView).text =
                intent.getStringExtra("subtitle")
        }
    }
}
